src/clear.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
```

Comment:
You have done great work on this project and all your functionalities work as they are supposed to. However, while disabling linters might seem convenient to get the work finished, this action may cause you to overlook certain errors that could cause major problems when you have to consider scalability. I suggest that you remove this line and manually fix these linter issues 👍